js/index.js

```diff
@@ -40,3 +40,70 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let navMenu = document.querySelector('nav');
+
+let newA1 = document.createElement('a');
+newA1.textContent = 'Cat Pictures';
+navMenu.appendChild(newA1);
+
+let navItems = document.querySelectorAll('nav > a')
```

Comment:
We went over several examples last night to make this way more robust and DRY. Do you see the benefit in using .forEach to do this?